.github/workflows/tests.yml

```diff
@@ -171,16 +171,22 @@ jobs:
 
   dist-tests:
     if: github.event.pull_request.draft == false
+    needs: [cpp-funcs]
     runs-on: ubuntu-latest
     steps:
       # --- Code update ---
       - name: "Check out code"
         uses: actions/checkout@v2
       - name: "Update faabric submodule"
         run: git submodule update --init faabric
-      # --- Build and test ---
+      # --- Set up and run ---
       - name: "Build distributed tests"
         run: ./dist-test/build.sh
+      - name: "Download CPP wasm"
+        uses: actions/download-artifact@v2
+        with:
+          name: cpp-wasm
+          path: ./dist-test/build/faasm-local/wasm
```

Comment:
This downloads the compiled wasm that we use in the normal tests into the distributed test container.